eth/api.go

```diff
@@ -715,6 +715,78 @@ func (api *ScrollAPI) GetNumSkippedTransactions(ctx context.Context) (uint64, er
 	return rawdb.ReadNumSkippedTransactions(api.eth.ChainDb()), nil
 }
 
+// FinalizedBlockHeight returns the L2 finalized block height.
+func (api *ScrollAPI) FinalizedBlockHeight(ctx context.Context) (uint64, error) {
```

Comment:
```suggestion
func (api *ScrollAPI) FinalizedBlockHeight(_ context.Context) (uint64, error) {
```

Reply:
changed in [0b95a6a](https://github.com/scroll-tech/go-ethereum/pull/548/commits/0b95a6a75227b5ab6b469947cfafcb81afe4babf)